meta-mender-raspberrypi/recipes-bsp/bootfiles/rpi-cmdline-mender.inc

```diff
@@ -1,2 +1,12 @@
 CMDLINE_ROOTFS:remove = "root=/dev/mmcblk0p2"
-CMDLINE_ROOTFS:append = " root=\${mender_kernel_root} "
```

Comment:
I couldn't find out how to escape either. I'm not sure it's possible.

But I have a simpler approach:

```
CMDLINE_ROOTFS:append = " root=${@'$%s' % '{mender_kernel_root}'} "
```

And btw: Why did this work previously? 🙄

Reply:
Ok. I see the problem. I think their upstream version is invalid actually. @TheYoctoJester said recently after talking to some OpenEmbedded people, that `${CMDLINE}` syntax is not valid within Python snippets, and has been working sort of by accident only in certain circumstances. I believe it should be changed to `d.getVar('CMDLINE')`.

So you could maybe attempt to upstream that. But I won't stand in the way of this PR, this is just a small and insignificant cleanup.